kv/db_test.go

```diff
@@ -267,35 +264,3 @@ func TestKVDBTransaction(t *testing.T) {
 		t.Errorf("expected value %q; got %q", value, gr.ValueBytes())
 	}
 }
-
-// TestHTTPAuthentication tests authentication for the KV http endpoint.
```

Comment:
This should also work by just changing from https to rpcs. `node` will succeed, `root` will not.